PWGDQ/TableProducer/tableMakerMC.cxx

```diff
@@ -410,6 +444,16 @@ struct TableMakerMC {
             continue;
           }
 
+          uint8_t dalitzMap = uint8_t(0);
+          if constexpr (static_cast<bool>(TTrackFillMap & VarManager::ObjTypes::DalitzBits)) {
+            for (auto& dalitz : dalitzBits) { // Look for the dalitz bits in dalitzBits table
+              if (dalitz.trackId() == track.globalIndex()) {
+                dalitzMap = dalitz.dalitzBits();
+                break;
+              }
+            }
+          }
```

Comment:
It is needed if we keep in the table only non-0 dalitzbits. Another solution for that could be to put all the dalitzbits in a std::map at the beginning of the process function (but this step could also take some time), and do the search from the std::map, since it has logarithmic complexity for search.

Reply:
It is needed if we keep in the table only non-0 dalitzbits. 
Another solution for that could be to put all the dalitzbits in a std::map at the beginning of the process function (but this step could also take some time), and do the search from the std::map, since it has logarithmic complexity for search. It would also remove the need for the DalitzBits argument in the fullSkimming function.